### ISSUE TITLE
Provide JsonUnitRequestMatchers for client-side rest testing

### ISSUE THREAD

**Issue by flowerrrr** [lukas-krecan/JsonUnit#185]
It would be nice, if json-unit-spring in addition to existing ResultMatcher would also provide a org.springframework.test.web.client.RequestMatcher, so that it is possible to match the json of POST-Requests when setting up Spring's MockRestServiceServer for client-side rest testing.

E.g.
```
mockServer = MockRestServiceServer.createServer(restTemplate);
mockServer.expect(requestTo("/sample"))
                  .andExpect(method(HttpMethod.POST))
                  .andExpect(json().isEqualTo(EXPECTED_JSON_REQUEST))   // <-- json() creates a RequestMatcher
                  .andRespond(withSuccess(jsonResponse, MediaType.APPLICATION_JSON_UTF8));
 ````

I would be willing to create a PR if the feature request is accepted.

**Comment by lukas-krecan** [comment]
Hi, sorry for the delay, I have been off for several weeks. If you do not have the PR already prepared, I can do it.

**Comment by lukas-krecan** [comment]
Released as 2.8.0
